src/main/java/com/aws/greengrass/security/SecurityService.java

```diff
@@ -307,15 +329,15 @@ public KeyManager[] getKeyManagers(URI privateKeyUri, URI certificateUri)
 
             try {
                 PrivateKey privateKey = keyPair.getPrivate();
-                List<X509Certificate> certificateChain =
-                        EncryptionUtils.loadX509Certificates(Paths.get(certificateUri));
+                List<X509Certificate> certificateChain = EncryptionUtils
+                        .loadX509Certificates(Paths.get(certificateUri));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Path traversal vulnerability detected. User-controlled input in file paths
can allow attackers to access files outside intended directories using
`../` sequences. Secure your code by using framework functions like
`getCanonicalPath()`, `normalize()`, or validating paths with
`.startsWith()` checks. Learn more:
https://cwe.mitre.org/data/definitions/22.html